### PR TITLE
SearchApplication implemented

### PR DESCRIPTION
**Pull request by JoePaul**
The main application for the program implemented

### CODE REVIEW

File: src/search/SearchApplication.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package search;
+
+public class SearchApplication {
+    public static void main(String[] args) {
+        Search.findPatternInFile(args[0], args[1]);
```
Comment (DavidJsson) [inline src/search/SearchApplication.java:5]: Specification mentions to have an input command to the program in the form of "search <pattern> <file>", consider implementing the search command.

Comment (DavidJsson) [inline src/search/SearchApplication.java:5]: You have no error handling for wrong inputs. Could be beneficial if there is some error handling for that.

Comment (JoePaul) [reply src/search/SearchApplication.java]: This is now fixed! Looked at the specification again and you are absolutely correct!

Comment (JoePaul) [reply src/search/SearchApplication.java]: This is now fixed! Looked at the specification again and you are absolutely correct!

File: src/search/SearchApplication.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package search;
+
+public class SearchApplication {
+    public static void main(String[] args) {
+        if (args[0].equalsIgnoreCase("search")) {
+            Search.findPatternInFile(args[1], args[2]);
```
Comment (DavidJsson) [inline src/search/SearchApplication.java:6]: If args[2] is missing the program will crash. Add some error handling for this.

**Review by DavidJsson — APPROVED** [review]

**Pull request merged**